tools/clang/lib/SPIRV/DeclResultIdMapper.cpp

```diff
@@ -398,6 +398,144 @@ SpirvLayoutRule getLayoutRuleForExternVar(QualType type,
   return SpirvLayoutRule::Void;
 }
 
+void handleImageFormat(const VarDecl *var, SpirvVariable *varInstr) {
+  const auto *imageFormatAttr = var->getAttr<VKImageFormatAttr>();
+  if (imageFormatAttr) {
```

Comment:
nit: `if (const auto *imageFormatAttr = var->getAttr<VKImageFormatAttr>()) {`

Reply:
Makes sense, I'll change that